pages/vote/elo.tsx

```diff
@@ -72,6 +88,12 @@ export default function Elo({ sessions }: EloProps): JSX.Element {
 
   async function sessionChoiceHandler(winningSession: EloSession, losingSession: EloSession, isDraw = false) {
     await postPair(winningSession.Id, losingSession.Id, isDraw)
+    logEvent('voting', 'vote', {
```

Comment:
Might be double-logging this event (see line 58 inside `postPair`)

Reply:
Good catch, have fixed that.